content/en/account_management/saml/okta.md

```diff
@@ -11,65 +9,123 @@ further_reading:
   text: "Configuring Teams & Organizations with Multiple Accounts"
 ---
 
+## Overview
+
+This page tells you how to set up the Datadog application in Okta. 
+
+Before proceeding, make sure that you are using the latest version of the Datadog application:
+1. In Okta, click **Applications**.
+1. Open the Datadog application.
+1. Select the **General** tab.
+1. Look for a field labeled **SSO Base URL**.
+
+{{< img src="account_management/saml/okta/sso_base_url.png" alt="Datadog application configuration in Okta, highlighting the SSO base URL" style="width:80%;" >}}
+
+If you can see the SSO Base URL, you are using the latest version and should follow the instructions on this page. If you don't see the SSO Base URL, configure Okta using the [legacy instructions][1].
```

Comment:
Good catch! I don't think very many people are using the legacy version. The reason Datadog came out with this updated version is because the legacy app doesn't work 😆 .